### PR TITLE
docs: add documentation for the assets directory

### PR DESCRIPTION
**Pull request by jvm-cognition**
## Please verify the following:

- [ ] `yarn test` **jest** tests pass with new tests, if relevant
- [x] `README.md` has been updated with your changes, if relevant

## Describe your PR
Updated the boilerplate README.md to include assets directory and how to use and organise them.

### CODE REVIEW

**Comment by frankcalise** [comment]
Nicely done @jvm-cognition - thanks for the contribution!

File: boilerplate/README.md [hunk 1]
```diff
@@ -110,6 +113,36 @@ This is a great place to put miscellaneous helpers and utilities. Things like da
...
+
+Ignite comes with a built-in `Icon` component. You can find detailed usage instructions in the [docs](https://github.com/infinitered/ignite/blob/master/docs/Components-Icon.md).
+
+**images**
+This is where your images will live, such as background images, logos, or any other graphics. You can use various formats such as PNG, JPEG, or GIF for your images.
+
+How to use your `icon` or `image` assets:
+
+```
+import { Image } from 'react-native';
```
Comment (frankcalise) [inline boilerplate/README.md:137]: Should we point to the `AutoImage` component from the components dir do you think?

**Review by frankcalise — COMMENTED** [review]
Looking good! Just a comment for some discussion

File: boilerplate/README.md (hunk 1 above)
Comment (jvm-cognition) [reply boilerplate/README.md:139]: Added AutoImage ref to docs

Comment (jvm-cognition) [reply boilerplate/README.md:139]: @frankcalise added the `AutoImage` ref in the docs

**Pull request merged**